src/docs/release/breaking-changes/network-policy-ios-android.md

```diff
@@ -60,14 +60,37 @@ your manifest would contain the following:
 </application>
 ```
 
-Furthermore:
+### Allowing cleartext connection for debug builds
+
+If you would like to allow HTTP connections for Android debug
+builds, you can add the following snippet to your $project_path\android\app\src\debug\AndroidManifest.xml:
+
+```
+<application android:usesCleartextTraffic="true"/>
+```
+
+For iOS, you can follow [these instructions](https://flutter.dev/docs/development/add-to-app/ios/project-setup#local-network-privacy-permissions) to create a `Info-debug.plist` and put this in:
+
+```
```

Comment:
I'm surprised this code snippet passed analysis without the `<!--skip-->` tag...